docs/types.get.md

```diff
@@ -200,8 +208,10 @@ the [can-define/list/list] will be updated with the `id`s of the `locations`.
 
 
 ```js
+import { DefineMap, DefineList } from "can";
+
 const Store = DefineMap.extend( {
-	locations: DefineList,
+	locations: { Default: DefineList },
```

Comment:
`locations: DefineList` is short-hand for `locations: { Type: DefineList }`.

Reply:
Fixed in commit 753b914.

Reply:
The new keyword was needed in front of `DefaultList`. Fixed in 73cc278.

Reply:
You don't need the `new` keyword with `Default`. The problem is just that this example doesn't work. I think your original change to `locations: { Default: DefineList }` is the correct way to do this. That is how you do what the description says

> The following example creates an empty locationIds can-define/list/list when a new instance of Store is created. 

Sorry to have led you down the wrong path.

Reply:
Reverted these changes made in 73cc278 in ae37c1a.